### PR TITLE
dev-python/Rx: add 4.0.3, update PYTHON_COMPAT for 3.2.0

### PR DESCRIPTION
**Pull request by LegoLivesMatter**
I have also changed the homepage URL's protocol from `http` to `https` as per `pkgcheck`'s recommendation.

Signed-off-by: Duje Mihanović <duje.mihanovic@skole.hr>

### CODE REVIEW

**Comment by telans** [comment]
@blueness could this please be reviewed and/or bumped to 4.0.4?

File: dev-python/Rx/Rx-4.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (AndrewAmmerlaan) [inline dev-python/Rx/Rx-4.0.3.ebuild:4]: Can we bump this to the latest `EAPI=8` please?

File: dev-python/Rx/Rx-4.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+MY_P="RxPY-${PV}"
+DESCRIPTION="Reactive Extensions for Python"
+HOMEPAGE="https://reactivex.io/"
+SRC_URI="
+	https://github.com/ReactiveX/RxPY/archive/v${PV}.tar.gz
+		-> ${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0"
+SLOT="4"
```
Comment (AndrewAmmerlaan) [inline dev-python/Rx/Rx-4.0.3.ebuild:20]: Have you confirmed that there is no file conflict between this `SLOT=4` and the older `SLOT=3` ebuilds?

**Comment by mgorny** [comment]
The package has been removed.

**Pull request closed without merging**